docker-compose.yml.latest

```diff
@@ -0,0 +1,29 @@
+version: '3'
```

Comment:
I'd invert the name into `docker-compose.latest.yml` since the file type is usually the last part?

Reply:
okay, updated